server/src/dbschema.js

```diff
@@ -1,8 +1,19 @@
 const db = require("./db");
 const Keygrip = require('keygrip');
+const pgpatcher = require("pg-patcher");
+
+// MAKE SURE TO SET THE PATCH LEVEL TO THE HIGHEST LEVEL
+// IN THE STATEMENT INSERT INTO property(key, value) VALUES('patch', 2);
 
 const createSQL = `
```

Comment:
I'm pretty sure we can't always unconditionally try to run createSQL - the challenges of creating indexes, for instance.  Also won't we unconditionally run `INSERT INTO property(key, value) VALUES('patch', 2);` ?